### ISSUE TITLE
Unable to remove secret chat buddy under pidgin.

### ISSUE THREAD

**Issue by tuxmaster** [majn/telegram-purple#175]
After start an secret chat and close it, the secret chat buddy (with the ! mark) exits twice in the buddy list. When I delete both and restart pidgin, then both will occur again.

**Comment by majn** [comment]
Which commit are you on?

**Comment by tuxmaster** [comment]
The last master one.

**Comment by majn** [comment]
https://github.com/majn/telegram-purple/issues/158 this issue is fixed, but the duplicated chats may still exist in your blist.

**Comment by tuxmaster** [comment]
In which version?
When I remove it, on the next restart the entry's show again.

**Comment by majn** [comment]
It was fixed in the following commit: https://github.com/majn/telegram-purple/commit/b10dc45bfc190f9939662475a0e4a8d36011e0f9

For further diagnosis it would be great if you could check if:
1. You use a commit later or equal to that.
2. If ~/.purple/blist.xml already contains duplicated nodes of secret chats that might have been created in previous versions. I've noticed that if nodes are duplicated, deleting a buddy will only delete one of those nodes and on restart the buddy will show up again, as long as there are still nodes left.

**Comment by tuxmaster** [comment]
I have the last version.
Then I have try to remove all from the ~/.purple/blist.xml  file.
But after restart pidgin all double entry's comes again.

**Comment by majn** [comment]
Just to clarify: The secret chats aren't being 'duplicated', they just show up again after deleting them, correct?

**Comment by tuxmaster** [comment]
Yes, after removing it from the config file, it only occurs twice.
And after removing one of from the pidgin gui both will removed.
I have make pictures to clarify it.
Close pidgin, and remove the duplicate from config file:
![1](https://cloud.githubusercontent.com/assets/6240474/11586017/705fd9d2-9a70-11e5-81d2-584419336583.png)
Then remove one of it via the gui and both will removed:
![2](https://cloud.githubusercontent.com/assets/6240474/11586092/f9b4f708-9a70-11e5-9271-8da0fcd43852.png)
Then close it and restart.
Now at the ui of pidgin the secret chat comes again, but only once.
![3](https://cloud.githubusercontent.com/assets/6240474/11586150/5ba7cc38-9a71-11e5-8d8a-c78923338202.png)
An now in the config file, it will be two times again.
